translations/fr.yaml

```diff
@@ -0,0 +1,247 @@
+    private-option-part-1: <strong>Privé:</strong> La vidéo ne peut être visionnée que par des chercheurs autorisés
+    private-option-list-with-databrary: Personnel de la plateforme Lookit, chercheurs travaillant avec {contact} sur l'étude "{name}" et utilisateurs autorisés de Databrary.
+    private-option-list-without-databrary: Personnel de la plateforme Lookit et chercheurs travaillant avec {contact} sur l'étude "{name}."
+    scientific-option: <strong>Scientifique et pédagogique:</strong> La vidéo pourrait être partagée à des fins scientifiques et pédagogiques. Par exemple, nous pourrions montrer un extrait vidéo dans le cadre d'une présentation lors d'un congrès scientifique ou lors d'un cours universitaire sur le développement cognitif, ou inclure une image ou une vidéo dans un article scientifique. Dans certains cas, un extrait vidéo ou des photos pourraient être accessibles sur Internet, par exemple, s'il s'agit de contenu supplémentaire pour un article scientifique.
+    publicity-option: <strong>Publicitaire:</strong> Veuillez choisir cette option si vous seriez enthousiaste à l'idée de voir votre enfant sur le site web Lookit ou dans un article de journal portant sur cette étude! Votre vidéo pourrait être partagée à des fins publicitaires, scientifiques ou pédagogiques; par contre, elle ne sera jamais partagée à des fins commerciales. Les extraits vidéos partagés pourraient être accessibles au grand public sur Internet.
+    withdrawal-header: Retrait des données vidéo
+    withdrawal-details: Chaque vidéo nous est utile, même si quelque chose s'est mal passé! Toutefois, si vous désirez supprimer votre vidéo {include_example, select, true { (par exemple, si à l'arrière-plan votre conjoint était en train de discuter de secrets d'État, etc.)} other {}}, cochez ici pour retirer entièrement vos données vidéo provenant de cette session de l'étude. Seul la vidéo servant à documenter votre consentement à l'étude sera conservée et celle-ci ne pourra être visionée que par le personnel de la plateforme Lookit, ainsi que par les chercheurs travaillant avec {contact} sur l'étude "{name}". Toute autre vidéo sera supprimée sans être visionnée.
+    feedback-label: Comment cela s'est-il passé? Avez-vous des suggestions pour améliorer cette étude?
+    withdrawal-confirmation: Vous avez choisi de supprimer vos données vidéo provenant de l'étude. Autre que votre consentement vidéo, toute vidéo sera supprimée sans être visionnée. Êtes-vous certain?
+    Withdraw: Retirer
+    facebook-share: Partagez cette étude sur Facebook!
+
+exp-lookit-video-assent:
+    header: Assentiment de l'enfant à participer
+    explanation-1: Pour les études avec des enfants plus âgés, nous devons vérifier que le parent <em>ainsi que</em> l'enfant acceptent de participer. <strong>Cette page est pour l'enfant!</strong>
+    explanation-2: Parents, veuillez aider votre enfant à lire et à naviguer, si nécessaire.
+    step-1: Renseignez-vous au sujet de cette étude

```

Comment:
line 43/44:
The download (Téléchargement) button covers part of the last word 'étude' of the prompt in line 43

Reply:
Moved this issue to #406. Thanks for the feedback!